frontend/src/components/ErrorBoundary/README.md

```
@@ -0,0 +1,225 @@
+  return (
+    <button onClick={resetError}>
+      Reset Application
+    </button>
+  );
+}
+```
+
+## Error Logging
+
+The error logger automatically:
+- Logs to console in development
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)